src/app/domains/wallets/domains.wallets.gateway.ts

```diff
@@ -1,46 +1,28 @@
 import { Injectable } from '@angular/core';
 import { Observable } from 'rxjs';
 import { IWalletApiResponse, IWalletPayload } from './domains.wallets.types';
-import { fakeRequest } from '../../common/http/common.http.fake-request';
-import { getRandomNumber } from 'src/app/common/utils/common.utils.random';
+import { HttpClient } from '@angular/common/http';
+import { API_WALLETS_URL } from './domains.wallets.constants';
 
 @Injectable({
   providedIn: 'root',
 })
 export class DomainsWalletsGateway {
+  private apiUrl: string = API_WALLETS_URL;
```

Comment:
I think you can just use in methods `API_WALLETS_URL`, you do not need this intermediate variable.